examples/da_commit/src/main.rs

```diff
@@ -80,7 +97,7 @@ async fn main() -> Result<()> {
     match load_commit_module_config::<ExtraConfig>() {
         Ok(config) => {
             info!(
-                module_id = config.id,
+                module_id = config.id.0,
```

Comment:
It's got `Display` implemented, but I stuck with my initial primitive approach to that, so it remained in. 😄

Just to make sure before I fix, you mean instead:

```rust
module_id = %config.id
```

right? Because I also tried reading into implementing `tracing::Value` for a custom type, which is required in order to trace the value without explicit display/debug modifiers (`%`/`?`), but from what I gathered it's intentionally designed to require explicit modifiers for custom types if you want to use their display/debug implementation. 😃